src/js/events/AppVersionsEvents.js

```diff
@@ -0,0 +1,9 @@
+var AppVersionsEvents = {
+  CHANGE: "APP_VERSIONS_EVENTS_CHANGE",
+  REQUEST_APP_VERSIONS: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSIONS",
+  REQUEST_APP_VERSIONS_ERROR: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSIONS_ERROR",
+  REQUEST_APP_VERSION: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSION",
+  REQUEST_APP_VERSION_ERROR: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSION_ERROR"
```

Comment:
Tiny nitpick, but can we get these constants a bit more concise? Eg `REQUEST_ONE` and `REQUEST_ALL` instead of `REQUEST_APP_VERSION` and `REQUEST_APP_VERSIONS`?